### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 14

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#190] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253455-a68af631-7290-4ec6-a070-932ae5b0ca3b.png)

**Comment by i0nut02** [comment]
<img width="364" alt="Schermata 2022-02-04 alle 23 10 14" src="https://user-images.githubusercontent.com/99051485/152610078-a278785d-4ae5-468d-91f9-074c287f396d.png">

**Comment by Exyss** [comment]
Stesso procedimento di @i0nut02

**Comment by LuginiAndrea** [comment]
Per doppio conteggio:
Ipotizziamo di scegliere una squadra di k giocatori fra n persone, con un solo portiere:

- O scegliamo prima i k giocatori fra n, e poi chi fra questi fa il portiere (parte a sinistra)

- O scegliamo prima il portiere fra le n persone, e poi i restanti k-1 giocatori di movimento fra le restanti n-1 persone
